### DISCUSSION TITLE
Death by 10,000 Nodes. Ideas to keep the tumbleweed tumblin'

### DISCUSSION THREAD

**Discussion by BuildBackBuehler** [Comfy-Org/ComfyUI-Manager#565] [Ideas]
I seriously hope there is some State of the ComfyUInion at some point soon. Would be appropriate with SD3 announced. 

It'd be sensible if there was some sorta aggregation of custom nodes with overlapping features and combining 'em to be either a. provided as a stable MeGaNoDe or b. (more sensibly) integrated into ComfyUI officially, deprecate a handful of nodes and send 'em to a graveyard channel. 

It'd make sense that once a feature has floated around for an iteration, that ComfyUI would integrate it officially as a "stable feature." 

But excuse the grandiosity, the grounded thought that came to mind was to expand/modify the flagging system. Therein highlighting nodes that are archived. I'd personally be a fan of also highlighting nodes that aren't actively maintained. Then we could filter out a ton of irrelevant noise. 

The red for failed could stay as-is. Though it may be best to reserve it for a different purpose and provide a textual/option based way of informing the user yo sh!p duck3d up yo! Optimally, "Install Custom Nodes + Install Missing Nodes" could be changed. Then you'd make it "Custom Nodes + Broken/Missing Nodes". Wam, bam, damn, provides the IMN button/interface way more purpose than it currently has. Could go as far as including unstable/experimental/can't-be-combined-with nodes that you have installed. The unstable part is pretty straightforward and could be taken as little/far as manpower can (error/warning logs in the interface)l. For the exp./can't-combine, what I mean is if X/Y/Z node is known to "not play friendly" with many nodes or it uses outlandish package versions that wouldn't mesh with your common node, they'd be there. Don't mean the Disco Diffusion or  Transformers-based Nodes...though they could/should be corralled together, too. But y'anyways, my point was that users can thus go into "Custom Nodes" and know that because they are using [Not Nice Node] now users would know beforehand which nodes will/won't work with their weirdo or when they'll need to put down the weirdo beforehand. 

Of course, originally, all I wanted to suggest was to flag nodes that the owner has archived by highlighting the node row with...grey? Whatever seems most appropriate. Figure that yellow is already for potential conflicts. Red is for broken ones. But those are my very late night thoughts, bye! Hope to stir some discussion and figure out how we'll avoid death by 10,000 knives before SD3 makes its way around. I guess nodes might not carryover much anyways. Just will be a repeat of SDXL's cycle of improvements/custom nodes/chaos, cheers!

**Comment by ltdrdata** [comment]
Although not with the announcement of SD3, improvements to the system for managing custom nodes are planned.
